__tests__/components/sections/communities/overview/scoreboard/_partials/FilterOption.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import FilterOption from "@/components/sections/communities/overview/scoreboard/_partials/FilterOption";
+
+describe("FilterOption", () => {
+  it("renders with label and is not checked by default", () => {
+    const label = "Test Label";
+    render(<FilterOption label={label} value="1" data="2" />);
+
+    expect(screen.getByLabelText(label)).toBeInTheDocument();
+    expect(screen.getByRole("radio")).not.toBeChecked();
+  });
```

Comment:
check if the text rendered is "Test Label"

Reply:
it is already tested, when checking if label is in the document.